nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-site-to-site/src/test/java/org/apache/nifi/remote/TestStandardRootGroupPort.java

```diff
@@ -38,7 +39,7 @@
 
 public class TestStandardRootGroupPort {
 
-    private RootGroupPort createRootGroupPort(NiFiProperties nifiProperties) {
+    private PublicPort createRootGroupPort(NiFiProperties nifiProperties) {
```

Comment:
how about changing this method name as well?

Reply:
Thanks @ekovacs good catch! Renamed.